### ISSUE TITLE
Update Crayfish-Commons dependency on symfony/security-core to allow 5.x

### ISSUE THREAD

**Issue by seth-shaw-unlv** [Islandora/documentation#1903]
The Drupal ldap module major version 4.0, the only version with Drupal 9 support, conflicts with symfony/security-core < 5 resulting in a conflict with crayfish-commons which requires either 3.4 or 4.4. This means that those using LDAP for authentication are stuck on Drupal 8 until this is resolved.

I haven't yet checked what we need to do to make crayfish-commons work with symfony/security-core:^5, if anything.

Tagging @elizoller as the current crayfish-commons maintainer.

**Comment by seth-shaw-unlv** [comment]
I went looking but couldn't find a good listing of security-core API changes....

**Comment by elizoller** [comment]
that ellipsis makes me think that you weren't successful

**Comment by seth-shaw-unlv** [comment]
Indeed, I wasn't.

**Comment by elizoller** [comment]
@seth-shaw-unlv can you give me more info on this one? what kind of environment are you running in? are you already on D9 in that env?
i spun up a playbook (d9) with ubuntu/focal64 and just ran `composer require 'drupal/ldap:^4.0'` and it succeeded with no trouble

**Comment by seth-shaw-unlv** [comment]
This was a D8 install with ldap 3 updating to D9. Perhaps I should spin up a fresh D8 Islandora, enable ldap 3, and try upgrading to D9 w/ ldap 4.

**Comment by kstapelfeldt** [comment]
Seth was updating an existing Islandora 1.0 site and had composer complain about this conflict. Eli tried to test and he was not able to reproduce. Where exactly is the hitch with the update progress?

**Comment by kstapelfeldt** [comment]
From @jordandukart https://packagist.org/packages/grahl/ldap - Eli pulled islandora-dev and Seth pulled in tagged version. Seth will try to reproduce.

**Comment by seth-shaw-unlv** [comment]
Okay. I took an isle-dc and modified the `make local` to give me a D8 and Islandora 1.1.1 before installing ldap:3.0.0-beta7. I then updated ldap to ^4 and nothing complained. I went the extra step and updated Islandora to ^2 and Drupal to ^9 and was able to get that to go.

So, can't reproduce. It must have been something weird I was doing with composer or my particular site. Closing as "can't reproduce".
